modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -21,7 +21,7 @@ The {ai-full} provides installation functionality as a service. This software-as
 ** Eliminates the need to install and run the {product-title} installer locally.
 ** Ensures the latest version of the installer up to the latest tested z-stream releases. Older versions remain available, if needed.
 ** Enables building automation by using the API without the need to run the {product-title} installer locally.
-* *Advanced networking:* The {ai-full} supports IPv4 networking with SDN and OVN, IPv6 and dual stack networking with OVN only, NMState-based static IP addressing, and an HTTP/S proxy. OVN is the default Container Network Interface (CNI) for OpenShift Container Platform 4.12 and later releases. SDN is supported up to {product-title} 4.14, but is not supported for {product-title} 4.15 and later releases.
+* *Advanced networking:* The {ai-full} supports IPv4 and IPv6 networking, as well as dual stack networking with the OVN-Kubernetes network plugin, NMState-based static IP addressing, and an HTTP/S proxy. OVN-Kubernetes is the default Container Network Interface (CNI) for OpenShift Container Platform 4.12 and later releases. OpenShift SDN is supported up to {product-title} 4.14, but is not supported for {product-title} 4.15 and later releases.
```

Comment:
```suggestion
* *Advanced networking:* The {ai-full} supports IPv4 and IPv6 networking, as well as dual-stack networking with the OVN-Kubernetes network plugin, NMState-based static IP addressing, and an HTTP/S proxy. OVN-Kubernetes is the default Container Network Interface (CNI) for {product-title} 4.12 and later releases. OpenShift SDN is supported up to {product-title} 4.14, but is not supported for {product-title} 4.15 and later releases.
```
I found some examples in the portal where 'dual stack' as an adjective is hyphenated, See: https://docs.redhat.com/en/documentation/red_hat_quay/3.8/html/manage_red_hat_quay/proc_manage-ipv6-dual-stack